src/pages/Earn/useFarmRegistry.ts

```diff
@@ -89,3 +94,20 @@ export const useFarmRegistry = () => {
 
   return farmSummaries
 }
+
+export const useUniqueBestFarms = () => {
+  const farmSummaries = useFarmRegistry()
+
+  const farmsUniqueByBestFarm = farmSummaries.reduce((prev: any, current) => {
```

Comment:
Don't use any.

The type is Record<string, FarmSummary>

You might be able to do `reduce<Record<string, FarmSummary>>`

Reply:
I used Record<string, FarmSummary> instead of any